### PR TITLE
Add mobile-friendly menu

### PR DESCRIPTION
**Pull request by gravdanger510**
screenshot 2017-03-07 15 57 48

### CODE REVIEW

File: app/assets/images/hamburger.svg [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<svg width="24" height="18" viewBox="0 0 24 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:figma="http://www.figma.com/figma/ns">
+<title>hamburger</title>
+<desc>Created using Figma</desc>
```
Comment (morgane) [inline app/assets/images/hamburger.svg]: could clean this up and remove tags like this one

File: app/views/application/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<header class="site-header">
+  <%= link_to image_tag("lxd-logo.svg", class: "lxd-logo"), "/" %>
+  <button class="mobile-nav-toggle" data-role="mobile-nav-toggle">
+    <%= link_to image_tag("hamburger.svg", class: "mobile-menu-icon"), "/" %>
+  </button>
+  <nav class="nav">
+    <button class="nav-item nav-close" data-role="mobile-nav-toggle">
+      &#10006
```
Comment (morgane) [inline app/views/application/_header.html.erb]: ?

File: app/views/application/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<header class="site-header">
```
Comment (morgane) [inline app/views/application/_header.html.erb]: why .erb and not .haml?

File: app/views/application/_header.html.erb (hunk 2 above)
Comment (gravdanger510) [inline app/views/application/_header.html.erb]: That's the close icon

File: app/views/application/_header.html.erb (hunk 3 above)
Comment (gravdanger510) [inline app/views/application/_header.html.erb]: Gabe recommended using erb since he hates haml and the client is using erb

**Pull request merged**